### ISSUE TITLE
NuxtError is not defined on initial project scaffold w/ ChakraUI, Universal, and Static

### ISSUE THREAD

**Issue by JessicaSachs** [nuxt/nuxt.js#8093]
### Versions

- nuxt: 2.14.15
- create-nuxt-app: create-nuxt-app/3.3.0 darwin-x64 node-v12.16.3
- node: 12.16.3, 10.22.0

### Reproduction

[Minimum reproducible example](https://github.com/JessicaSachs/chakra-nuxt-static-example)

### Steps to reproduce

1. Go into the newly created project
2. Run `yarn dev`
3. Navigate to your app
4. See that the console throws an error :-(

### What is Expected?

That it works fine :-)

### What is actually happening?

There's an error.

```
 ERROR  [Vue warn]: Error in callback for watcher "nuxt.err": "ReferenceError: NuxtError is not defined"

(found in <Root>)


 ERROR  [Vue warn]: Error in nextTick: "ReferenceError: NuxtError is not defined"


 ERROR  NuxtError is not defined                                          08:27:43

  at Vue.errorChanged (server.js:290:27)
  at Watcher.run (node_modules/vue/dist/vue.runtime.common.dev.js:4554:19)
  at flushSchedulerQueue (node_modules/vue/dist/vue.runtime.common.dev.js:4298:13)
  at Array.<anonymous> (node_modules/vue/dist/vue.runtime.common.dev.js:1976:12)
  at flushCallbacks (node_modules/vue/dist/vue.runtime.common.dev.js:1902:14)
```

**Comment by JessicaSachs** [comment]
Ah, it's only for ChakraUI + Nuxt.. Going through and creating a minimum reproducible example.

**Comment by pi0** [comment]
Hi @JessicaSachs. This issue should be fixed with [2.14.6](https://github.com/nuxt/nuxt.js/releases/tag/v2.14.6)
